src/dash/Dashboard.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import HomeMenuBar from '../MenuBar.jsx';
+import DefaultDash from './DefaultDash.jsx';
+import { Provider } from 'react-redux';
+import store from '../redux/store.js';
+import AboutUs from '../home/AboutUs.jsx';
+import {Router, Route, browserHistory} from 'react-router';
+
+
+
+const DashComp = function() {
+  return (
+    <div>
+      <HomeMenuBar homeordash='dash'/>
+      <Router history={browserHistory}>
+          <Route path="/dash" component={DefaultDash}/>
```

Comment:
this needs to have an `exact` prop. Otherwise it'll render both components when you navigate to `dash/about`. React router "falls through".